mypy/build.py

```diff
@@ -1015,6 +1023,36 @@ def __init__(self,
             # Parse the file (and then some) to get the dependencies.
             self.parse_file()
 
+    def skipping_ancestor(self, id: str, path: str, ancestor_for: 'State') -> None:
+        # TODO: Read the path (the __init__.py file) and return
+        # immediately if it's empty or only contains comments.
+        # But beware, some package may be the ancestor of many modules,
+        # so we'd need to cache the decision.
+        manager = self.manager
+        manager.errors.set_import_context([])
+        manager.errors.set_file(ancestor_for.xpath)
+        manager.errors.report(-1, "Ancestor package '%s' silently ignored" % (id,),
+                              severity='note', only_once=True)
```

Comment:
I did -- if a package has many children it would be annoying to hear about this each time. And there's no possibility that some of the children have their ancestor ignored while others don't -- it's one ancestor and it's either ignored or not.